components/airtable/sources/new-or-modified-records-in-view/new-or-modified-records-in-view.js

```diff
@@ -1,24 +1,19 @@
-const airtable = require('../../airtable.app.js')
 const moment = require('moment')
 const axios = require('axios')
 
+const common = require('../common')
+
 module.exports = {
+  ...common,
   name: "New or Modified Records in View",
   description: "Emit an event for each new or modified record in a view",
   key: 'airtable-new-or-modified-records-in-view',
-  version: "0.0.2",
+  version: "0.0.3",
   props: {
-    db: "$.service.db",
-    airtable,
+    ...common.props,
     baseId: { type: "$.airtable.baseId", appProp: "airtable" },
```

Comment:
Does it make sense to include `baseId` in the common props or was there a reason you left it out?

Reply:
It does make sense actually, looks like Airtable uses this "base" concept for every product, so it makes sense to have it in `common`. I'll move it there.